get

```diff
@@ -71,24 +71,34 @@ initDownloadTool() {
 		DOWNLOAD_TOOL="curl"
 	elif type "wget" > /dev/null; then
 		DOWNLOAD_TOOL="wget"
-	else 
+	else
 		fail "You need curl or wget as download tool. Please install it first before continue"
 	fi
 	echo "Using $DOWNLOAD_TOOL as download tool"
 }
 
+initGithubAuth() {
+	if [ ! -z "$GITHUB_USER$GITHUB_TOKEN" ]; then
```

Comment:
What if `$GITHUB_USER` has a value and `$GITHUB_TOKEN` is empty? When using hub there are cases where someone could supply `$GITHUB_PASSWORD` instead.

These two values should be checked separately to make sure both are not empty.

Reply:
The GitHub token is used in the same way as the password, so there's no need for a separate env var. You would use the password in place of the token. 

That said, I'm going to look at changing the PR as per the discussion above so we don't hit the API endpoint except in the event of an error, so those vars won't be needed to install. If an error occurs, the rate-limiting may still be an issue, but the problem becomes "we can't provide a more informative error because we're rate limited" instead of "we can't install because we're rate limited". In that case, the GitHub may be required and in that event, your suggestion would be reasonable to split the check out.